DEPS

```diff
@@ -155,7 +155,7 @@ deps = {
    'https://boringssl.googlesource.com/boringssl.git' + '@' + Var('dart_boringssl_rev'),
 
   'src/third_party/dart':
-   Var('dart_git') + '/sdk.git' + '@' + Var('dart_revision'),
+  'https://github.com/dart-lang/sdk.git' + '@' + Var('dart_revision'),
```

Comment:
This needs to go back to `Var('dart_git')`

Reply:
We need to actually land https://dart-review.googlesource.com/c/sdk/+/151525, and then update this PR to the hash that comes out on dart/sdk master. Waiting for Flutter to turn green to do that.